doc/classes/Color.xml

```diff
@@ -241,38 +244,41 @@
 			<return type="bool" />
 			<param index="0" name="color" type="String" />
 			<description>
-				Returns [code]true[/code] if [param color] is a valid HTML hexadecimal color string. [param color] is not case sensitive, and may be prefixed with a '#' character.
-				For a string to be valid it must be three-digit or six-digit hexadecimal, and may contain an alpha channel value.
+				Returns [code]true[/code] if [param color] is a valid HTML hexadecimal color string. The string must be a hexadecimal value (case-insensitive) of either 3, 4, 6 or 8 digits, and may be prefixed by a hash sign ([code]#[/code]). This method is identical to [method String.is_valid_html_color].
 				[codeblocks]
 				[gdscript]
-				var result = Color.html_is_valid("#55aaFF") # result is true
-				result = Color.html_is_valid("#55AAFF20") # result is true
-				result = Color.html_is_valid("55AAFF") # result is true
-				result = Color.html_is_valid("#F2C") # result is true
-				result = Color.html_is_valid("#AABBC) # result is false
-				result = Color.html_is_valid("#55aaFF5") # result is false
+				Color.html_is_valid("#55aaFF")   # Returns true
+				Color.html_is_valid("#55AAFF20") # Returns true
+				Color.html_is_valid("55AAFF")    # Returns true
+				Color.html_is_valid("#F2C")      # Returns true
+
+				Color.html_is_valid("#AABBC)     # Returns false
+				Color.html_is_valid("#55aaFF5")  # Returns false
 				[/gdscript]
 				[csharp]
-				var result = Color.HtmlIsValid("#55AAFF"); // result is true
-				result = Color.HtmlIsValid("#55AAFF20"); // result is true
-				result = Color.HtmlIsValid("55AAFF); // result is true
-				result = Color.HtmlIsValid("#F2C"); // result is true
-				result = Color.HtmlIsValid("#AABBC"); // result is false
-				result = Color.HtmlIsValid("#55aaFF5"); // result is false
+				Color.HtmlIsValid("#55AAFF");   // Returns true
+				Color.HtmlIsValid("#55AAFF20"); // Returns true
+				Color.HtmlIsValid("55AAFF);     // Returns true
+				Color.HtmlIsValid("#F2C");      // Returns true
+
+				Color.HtmlIsValid("#AABBC");    // Returns false
+				Color.HtmlIsValid("#55aaFF5");  // Returns false
 				[/csharp]
 				[/codeblocks]
 			</description>
 		</method>
 		<method name="inverted" qualifiers="const">
 			<return type="Color" />
 			<description>
-				Returns the inverted color [code](1 - r, 1 - g, 1 - b, a)[/code].
+				Returns the color with its [member r], [member g], and [member b] components inverted ([code](1 - r, 1 - g, 1 - b, a)[/code]).
 				[codeblocks]
 				[gdscript]
+				var black = Color.WHITE.inverted()
 				var color = Color(0.3, 0.4, 0.9)
 				var inverted_color = color.inverted() # Equivalent to `Color(0.7, 0.6, 0.1)`
 				[/gdscript]
 				[csharp]
+				var black = Color.WHITE.inverted()
```

Comment:
C# constants are PascalCase and they are exposed in the `Colors` static class (note the `s` at the end).

```suggestion
				var black = Colors.White.Inverted();
```

Reply:
You've fixed the constant name but missed the capitalization of the `Inverted` method and missing semicolon.